app/services/status-messages.ts

```diff
@@ -0,0 +1,61 @@
+import { service } from '@ember-decorators/service';
+import Service from '@ember/service';
+import config from 'ember-get-config';
+
+export interface StatusMessage {
+    id: string;
+    jumbo?: boolean;
+    dismiss: boolean;
+    extra?: any;
+    class: string;
+}
+
+export default class StatusMessages extends Service {
+    @service cookies;
+
+    messages: StatusMessage[];
+    nextMessages: StatusMessage[];
+
+    constructor() {
+        super();
+        this.nextMessages = this._getCookieMessages();
+    }
+
+    _getCookieMessages(this: StatusMessages): StatusMessage[] {
+        const cookies = this.get('cookies');
+        if (!cookies.read(config.OSF.statusCookie)) {
+            return [];
+        }
+        // decode commas and \\
+        let status = cookies.read(config.OSF.statusCookie).replace(/\\054/g, ',').replace(/\\/g, '');
+        // remove outside quotes
+        status = status.slice(1).slice(0, -1);
```

Comment:
How about `.slice(1, -1)` and after .read() so you can make it a const?

Reply:
Dont think I can make it a constant because then linter complains about not chaining slice with other methods, but will clean-up the slice!

Reply:
`/^"|/|"$/g` in the last .replace would work too